### PR TITLE
DOC-239: Rates JSDoc

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Namespaces:
- Rates

Case: https://highfidelity.atlassian.net/browse/DOC-239

Note: No executable code changes.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22791/HighFidelity-Beta-PR16564-73ad5d1.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22791/HighFidelity-Beta-Interface-PR16564-73ad5d1.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22791/HighFidelity-Beta-PR16564-73ad5d1.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22791/HighFidelity-Quest-Beta-PR16564-73ad5d1.apk)

File: interface/src/scripting/RatesScriptingInterface.h [hunk 1]
```diff
@@ -14,6 +14,41 @@
...
+ *
+ * @namespace Rates
+ *
+ * @hifi-interface
+ * @hifi-client-entity
+ * @hifi-avatar
+ *
+ * @property {number} render - The rate at which new GPU frames are being created, in Hz.
+ *     <em>Read-only.</em>
+ * @property {number} present - The rate at which the display plugin is presenting to the display device, in Hz
```
Comment (jennaingersoll) [inline interface/src/scripting/RatesScriptingInterface.h]:
```suggestion
 * @property {number} present - The rate at which the display plugin is presenting to the display device, in Hz.
```

Comment (ctrlaltdavid) [reply interface/src/scripting/RatesScriptingInterface.h]: Fixed.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22799/HighFidelity-Beta-PR16564-4c1f746.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22799/HighFidelity-Beta-Interface-PR16564-4c1f746.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22799/HighFidelity-Beta-PR16564-4c1f746.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22799/HighFidelity-Quest-Beta-PR16564-4c1f746.apk)

**Pull request merged**